### ISSUE TITLE
Insufficient privileges for Windows accessibility library

### ISSUE THREAD

**Issue by hrj** [UprootLabs/gngr#212]
Continuing from #171 
cc @rugglcon

The exception is:
```
Aug 09, 2016 1:28:17 PM org.lobobrowser.main.ReuseManager launch
WARNING: Another instance of the application must have been running but was not shut down properly.
Deleting residues of the last instance and creating a new instance.
Codesource: file:/C:/Program%20Files/Java/jre1.8.0_91/lib/ext/access-bridge-64.jar
Returning permissions: java.security.Permissions@497470e (
("java.security.SecurityPermission" "putProviderProperty.")
("java.lang.RuntimePermission" "accessClassInPackage.")
("java.lang.RuntimePermission" "loadLibrary.*")
("java.io.FilePermission" "C:\Program Files\Java\jre1.8.0_91-" "read,execute")
)

Exception in thread "main" java.lang.ExceptionInInitializerError
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at java.lang.Class.newInstance(Unknown Source)
at java.awt.Toolkit.loadAssistiveTechnologies(Unknown Source)
at java.awt.Toolkit.getDefaultToolkit(Unknown Source)
at javax.swing.UIManager.(Unknown Source)
at org.lobobrowser.main.PlatformInit.initLookAndFeel(PlatformInit.java:171)
at org.lobobrowser.main.PlatformInit.init(PlatformInit.java:325)
at org.lobobrowser.main.ReuseManager.launch(ReuseManager.java:159)
at org.lobobrowser.main.EntryPoint.launch(EntryPoint.java:63)
at org.lobobrowser.main.EntryPoint.lambda$0(EntryPoint.java:79)
at java.security.AccessController.doPrivileged(Native Method)
at org.lobobrowser.main.EntryPoint.privilegedLaunch(EntryPoint.java:78)
at org.lobobrowser.main.EntryPoint.main(EntryPoint.java:56)
Caused by: java.security.AccessControlException: access denied ("java.io.FilePermission" "C:\Program%20Files\Java\jre1.8.0_91\lib\ext\amd64\JavaAccessBridge-64.dll" "read")
at java.security.AccessControlContext.checkPermission(Unknown Source)
at java.security.AccessController.checkPermission(Unknown Source)
at java.lang.SecurityManager.checkPermission(Unknown Source)
at java.lang.SecurityManager.checkRead(Unknown Source)
at java.io.File.exists(Unknown Source)
at sun.misc.Launcher$ExtClassLoader.findLibrary(Unknown Source)
at java.lang.ClassLoader.loadLibrary(Unknown Source)
at java.lang.Runtime.loadLibrary0(Unknown Source)
at java.lang.System.loadLibrary(Unknown Source)
at com.sun.java.accessibility.AccessBridgeLoader$1.run(AccessBridgeLoader.java:38)
at java.security.AccessController.doPrivileged(Native Method)
at java.security.AccessController.doPrivileged(Unknown Source)
at com.sun.java.accessibility.AccessBridgeLoader.(AccessBridgeLoader.java:35)
... 16 more
```

**Comment by hrj** [comment]
@ruggIcon please mention your Windows OS version.

I will try to debug this tomorrow. In the meanwhile, if you wish, you may try to fix it yourself.

The security policy is defined in this class: `org.lobobrowser.security.LocalSecurityPolicy` in PlatformCore project. Can you try poking that class a bit? The lines from 432 to 446 are relevant as they deal with the JDK classes.

----

Just noticed that you are gonna try it on Linux. That would do too :)

**Comment by hrj** [comment]
cc @rugglcon again.

(I had typos in your id in the previous comments).

**Comment by rugglcon** [comment]
Yeah I can look around to see if I can add anything useful and I'm using Windows 10 when I get these exceptions.

**Comment by rugglcon** [comment]
So it seems to be coming from this line in the errors: 
`Caused by: java.security.AccessControlException: access denied ("java.io.FilePermission" "C:\Program%20Files\Java\jre1.8.0_91\lib\ext\amd64\JavaAccessBridge-64.dll" "read")`
I tested adding an explicit file permission to this location and it still threw this error. This file doesn't exactly exist, there's just .jar files in the "ext" directory. If you've got any further suggestions let me know, but even explicitly adding that path as a file permission didn't fix the issue.
![error](https://cloud.githubusercontent.com/assets/14317362/17532583/ea817924-5e46-11e6-8928-e924aba69a85.png)

**Comment by rugglcon** [comment]
I did some digging, and tried it on a different Windows machine. When I tried to run gngrDev earlier I was not logged into the administrator account of that computer and so it was actually the user account that didn't have permissions to read those files, it wasn't the security policies in gngr. 

I guess I'm not entirely sure if there's anything to change for that, but I can get the browser up and running now.

**Comment by hrj** [comment]
@rugglcon Rather belated, but thanks for the feedback you provided while getting gngr working. I must have forgotten to comment at the time.
